binder/src/main/java/io/grpc/binder/BinderServerBuilder.java

```diff
@@ -48,7 +47,6 @@
 /**
  * Builder for a server that services requests from an Android Service.
  */
-@ExperimentalApi("https://github.com/grpc/grpc-java/issues/8022")
 public final class BinderServerBuilder
     extends ForwardingServerBuilder<BinderServerBuilder> {
 
```

Comment:
`enableStats()` and `enableTracing()` need to be marked experimental or internal or such. We will not want them stabilized.

Reply:
Marked as experimental.